file-formats/sajc/type/zif_aff_sajc_v1.intf.abap

```diff
@@ -0,0 +1,40 @@
+INTERFACE zif_aff_sajc_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Header for SACJ Objects</p>
+    BEGIN OF ty_header,
+      "! <p class="shorttext">Description</p>
+      "! Description of the ABAP object
+      "! $required
+      description           TYPE c LENGTH 120,
+      "! <p class="shorttext">Original Language</p>
+      "! Original language of the development object.
+      "! $required
+      original_language     TYPE sy-langu,
+      "! $required
+      "! $values {@link zif_aff_types_v1.data:co_abap_language_version_cloud}
+      abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version,
```

Comment:
@wurzka is the language version required, or it this bad copy&pasting of source code?

Reply:
Language version is not required. But it should have a default